src/earn/EarnEnterAmount.tsx

```diff
@@ -238,9 +279,10 @@ function EarnEnterAmount({ route }: Props) {
 
   const disabled =
     // Should disable if the user enters 0, has enough balance but the transaction is not possible, or does not have enough balance
-    !!tokenAmount?.isZero() || !transactionIsPossible
+    !!tokenAmount?.isZero() || (!isWithdrawal && !transactionIsPossible)
```

Comment:
So continue is always enabled for any non-zero value?

Reply:
Fixed in 584acfbe3dadf0405ebdee8ef29192232de4011a! This was related to the mix-up in https://github.com/valora-inc/wallet/pull/6128#discussion_r1796081650.